crates/polars-python/src/lazyframe/visitor/expr_nodes.rs

```diff
@@ -1272,7 +1286,15 @@ pub(crate) fn into_py(py: Python<'_>, expr: &AExpr) -> PyResult<PyObject> {
                 FunctionExpr::Log1p => ("log1p",).into_py_any(py),
                 FunctionExpr::Exp => ("exp",).into_py_any(py),
                 FunctionExpr::Unique(maintain_order) => ("unique", maintain_order).into_py_any(py),
-                FunctionExpr::Round { decimals } => ("round", decimals).into_py_any(py),
+                FunctionExpr::Round { decimals, mode } => (
+                    "round",
+                    decimals,
+                    match mode {
+                        RoundMode::HalfToEven => "half_to_even",
+                        RoundMode::HalfAwayFromZero => "half_away_from_zero",
+                    },
```

Comment:
Since the enum is annotated with `strum`, we can just use:
```
Into::<&str>::into(mode)
```
here, I think.

Reply:
Could you please bump the minor IR version in `visit.rs` (in the impl of `NodeTraverser`).